admin_pages/transactions/templates/txn_admin_details_main_meta_box_txn_details.template.php

```diff
@@ -302,6 +302,12 @@ class="admin-primary-mbox-total-tr<?php echo $pay_totals_class; ?>">
                         <?php esc_html_e('Apply Refund', 'event_espresso'); ?>
                     </a>
                 </li>
+                <?php
+                /**
+                 * Allows extend the fields at actions area.
+                 */
+                do_action('AHEE__txn_admin_details__after_actions_buttons');
```

Comment:
this should use the full template name in the filter name, so:

```suggestion
                do_action('AHEE__txn_admin_details_main_meta_box_txn_details__after_actions_buttons');
```

Reply:
Awesome! Thanks for the suggestion 👍